spec/lib/vmdb/settings_spec.rb

```diff
@@ -202,4 +214,25 @@
 
     include_examples "password handling"
   end
+
+  describe ".for_resource" do
+    let(:server) { FactoryGirl.create(:miq_server) }
+
+    it "without database changes" do
+      settings = Vmdb::Settings.for_resource(server)
+      expect(settings.api.token_ttl).to eq "10.minutes"
+    end
+
+    it "with database changes" do
+      server.settings_changes.create!(:key => "/api/token_ttl", :value => "2.minutes")
+      settings = Vmdb::Settings.for_resource(server)
+      expect(settings.api.token_ttl).to eq "2.minutes"
+    end
+
+    it "with database changes on an Array" do
+      server.settings_changes.create!(:key => "/log/collection/current/pattern", :value => ["*.log"])
```

Comment:
Note, I ask this because I didn't originally realize what this PR was doing since I couldn't see that the existing value was an array we want to replace instead of merge.